Update CSS token per new design system theming capability

irma-kurnia-alaska opened the pull request:
# Alaska Airlines Pull Request

**Fixes:** #29 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.


## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability


## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

irma-kurnia-alaska committed:
perf(token): update token per new theming #44

Change to be committed:
modified: package-lock.json
modified: package.json
modified: demo/index.html
modified: src/style.scss
modified: src/style-toaster.scss
modified: README.md

blackfalcon commented on demo/index.html:
```diff
@@ -23,6 +23,7 @@
       href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
     />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```
Please update to mirror this code 

https://github.com/AlaskaAirlines/auro-avatar/blob/main/demo/index.html#L12-L17

irma-kurnia-alaska replied:
ok, sure. thank you.

blackfalcon commented on package.json:
```diff
@@ -25,13 +25,15 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
```
Remove legacy package reference

blackfalcon commented on package.json:
```diff
@@ -25,13 +25,15 @@
+    "@aurodesignsystem/webcorestylesheets": "^5.0.2"
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.14.0",
```
Remove legacy package reference

blackfalcon commented on src/style-toaster.scss:
```diff
@@ -4,27 +4,30 @@
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties';
```
```suggestion
```

blackfalcon commented on src/style-toaster.scss:
```diff
@@ -4,27 +4,30 @@
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
```suggestion
```

Remove duplicate reference

blackfalcon commented on src/style.scss:
```diff
@@ -4,55 +4,58 @@
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties';
```
```suggestion
```

blackfalcon commented on src/style.scss:
```diff
@@ -4,55 +4,58 @@
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
```suggestion
```

blackfalcon reviewed (changes requested):
Please update all npm dependencies and update repo to support Node support policy https://auro.alaskaair.com/node-support#gsc.tab=0

irma-kurnia-alaska committed:
fix: feedback review for update library dependency and remove unused code

Change to be committed:
modified: package.json
modified: package-lock.json
modified: demo/index.html
modified: src/style.scss
modified: src/style-toaster.scss

blackfalcon commented:
:tada: This PR is included in version 1.0.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@aurodesignsystem/auro-toast/v/1.0.1)
- [GitHub release](https://github.com/AlaskaAirlines/auro-toast/releases/tag/v1.0.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
